pkgs/by-name/zo/zoraxy/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  ...
+}:
+buildGoModule rec {
+  pname = "zoraxy";
+  version = "3.0.4";
+  src = fetchFromGitHub {
+    owner = "tobychui";
+    repo = "zoraxy";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-tBEIePauoZHs7aaVWCr9l8nvMT9YUDuUK9oTh7iwSRw=";
+  };
+
+  sourceRoot = "${src.name}/src";
+
+  vendorHash = "sha256-YI6LSccPDnVhGyPIEFIF41ex0WJlHtb3nP+8+1G/LA0=";
+
+  # For some reason the acme_test.go references a stackoverflow.pem file not present in the repository
```

Comment:
Can we only ignore that one test?

Reply:
i don't really know how to ignore a single test. Do you have some docs where I can look up?